### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by jhlivingston**
Created dependabot file to be run daily with Kristen and Nico as default reviewers.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    versioning-strategy: increase
+    reviewers:
+      - "KristenKatona"
+      - "nicogonza"
+    ignore:
+      - dependency-name: "@types/node"
+        versions:
+          - ">= 10"
+  - package-ecosystem: "github-actions"
+     open-pull-requests-limit: 10
```
Comment (ColgateLeoAscenzi) [inline .github/dependabot.yml:19]: Just needs to remove this extra space for it to function

**Review by ColgateLeoAscenzi — CHANGES_REQUESTED** [review]
Looks good to me except for an extra space on line 19, after that's fixed I think you can merge it, but maybe check with @KristenKatona ?

**Comment by jhlivingston** [comment]
The extra space is now removed. @ColgateLeoAscenzi & @KristenKatona Can you approve requested changes?

**Review by ColgateLeoAscenzi — APPROVED** [review]
Yep, looks good!

**Pull request merged**